syscall/devices/hostsock/hostsock.c

```diff
@@ -271,6 +271,14 @@ static oe_fd_t* _hostsock_accept(
             OE_RAISE_ERRNO_MSG(oe_errno, "retval=%d", retval);
 
         new_sock->host_fd = retval;
+
+        // copy peer addr to out buffer
+        if (addrlen && *addrlen <= addrlen_in)
```

Comment:
Isn't the `*addrlen <= addrlen_in` check redundant? memcpy_s already checks the sizes.

Reply:
Yes, that's right. I removed it.